recipes/qpoases/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import glob
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ConanRecipe(ConanFile):
+    name = "qpoases"
+
+    description = "Open-source C++ implementation of the recently proposed online active set strategy."
+    topics = ("conan", "container", "parametric", "quadratic", "programming")
+
+    homepage = "https://github.com/coin-or/qpOASES"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    license = "LGPL-2.1"
+
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "shared": [False],
+    }
+    default_options = {"fPIC": True, "shared": False}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("qpOASES-*/")[0]
```

Comment:
Future not when the project has release we can use `self.name + "-" + self.version` to save the import of glob

Reply:
Yes, currently the extracted dir looks like this: `qpOASES-releases-3.2.1` :-/

Reply:
done